volatility3/framework/plugins/windows/debugregisters.py

```diff
@@ -24,7 +24,7 @@
 class DebugRegisters(interfaces.plugins.PluginInterface):
     # version 2.6.0 adds support for scanning for 'Ethread' structures by pool tags
     _required_framework_version = (2, 6, 0)
-    _version = (1, 0, 1)
+    _version = (1, 0, 2)
```

Comment:
So neither the version, not the threads dependency need changing here, since you're not using a thread of the newer version.